integration-cli/docker_cli_images_test.go

```diff
@@ -288,3 +290,22 @@ func (s *DockerSuite) TestImagesFormatDefaultFormat(c *check.C) {
 	out, _ = dockerCmd(c, "--config", d, "images", "-q", "myimage")
 	c.Assert(out, checker.Equals, imageID+"\n", check.Commentf("Expected to print only the image id, got %v\n", out))
 }
+
+func (s *DockerSuite) TestImagePullWrongCreds(c *check.C) {
```

Comment:
s/`DockerSuite`/`DockerRegistryAuthSuite`/

you always get connection refused in the test otherwise (because nothing listen on `privateRegistryURL`) and this is why @icecrime and CI have that error